engine/src/conversion/error_reporter.rs

```diff
@@ -205,10 +203,14 @@ pub(crate) fn convert_item_apis<F, A, B: 'static>(
     }))
 }
 
-fn ignored_item<A: AnalysisPhase>(ns: &Namespace, ctx: ErrorContext, err: ConvertError) -> Api<A> {
-    Api::IgnoredItem {
-        name: ApiName::new(ns, ctx.get_id().clone()),
+fn ignored_item<A: AnalysisPhase>(
```

Comment:
Is this bit of the change still necessary if you're now always generating an `IgnoredItem`, per previous review comment?

Reply:
Yes, because there's still no `Api::IgnoredItem` for those, because that requires a `name` which doesn't exist. Do you see a reason to create an `Api` for them, and if so would you prefer `Api::IgnoredItem` having `name: Option<ApiName>` or a new `Api::SilentlyIgnoredItem`?

Reply:
OK no worries then. Let's stick with it as is.